src/lib/util.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var $ = require('$jquery');
```

Comment:
Just out of interest why do we use `jquery` in the `HUD` seems like a huge dependency for a small script that runs on the `users` side?